packages/lib/constants.ts

```diff
@@ -20,6 +20,7 @@ export const DEFAULT_BRAND_COLOR = "#64748b";
 export const FB_LOGO_URL =
   "https://s3.eu-central-1.amazonaws.com/listmonk-formbricks/Formbricks-Light-transparent.png";
 
+export const IS_LOCALHOST = ["localhost", "127.0.0.1", "::1"].includes(new URL(WEBAPP_URL).hostname);
```

Comment:
Nodejs provides the `NODE_ENV` variable which is either `development` or `production`. I think this is the much better choice here, because it's also possible to run production Formbricks on localhost depending on the proxy setup.